corehq/apps/cloudcare/views.py

```diff
@@ -131,9 +131,6 @@ def get_web_apps_available_to_user(self, domain, user):
         except DomainMembershipError:
```

Comment:
Should we completely remove `role` from this function? 
I am not sure if it's serving any purpose here?

Reply:
1d8a8737e8f